packages/ui/src/components/va-app-bar/VaAppBar.demo.vue

```diff
@@ -144,7 +144,7 @@ export default {
 
 <style>
 .frame {
-  border-radius: 1rem;
+  border-radius: var(--va-border-radius);
```

Comment:
Doesn't seem to be declared anywhere, so unlikely to work.
`--va-border-radius`